integration_tests/src/payments_althea.rs

```diff
@@ -1,2 +1,125 @@
+
+pub fn get_althea_evm_priv() -> EthermintPrivateKey {
+    ALTHEA_EVM_PRIV_BYTES
+        .parse::<clarity::PrivateKey>()
+        .unwrap()
+        .into()
+}
+
+pub fn get_althea_evm_pub() -> AltheaAddress {
+    get_althea_evm_priv().to_address(ALTHEA_PREFIX).unwrap()
+}
+
+pub fn get_althea_grpc() -> String {
+    format!("http://{}:9091", NODE_IP)
+}
+
+pub async fn run_althea_payments_test_scenario() {
+    info!("Starting althea payments test");
+
+    let node_config = five_node_config();
+    let namespaces = node_config.0;
+    let expected_routes = node_config.1;
+
+    let (rita_settings, rita_exit_settings) = get_default_settings();
+
+    namespaces.validate();
+    start_postgres();
+
+    let res = setup_ns(namespaces.clone());
+    info!("Namespaces setup: {res:?}");
+
+    // Modify configs to use Althea chain
+    let rita_exit_settings = althea_system_chain_exit(rita_exit_settings);
+    let rita_settings = althea_system_chain_client(rita_settings);
+
+    let rita_identities = thread_spawner(namespaces.clone(), rita_settings, rita_exit_settings)
+        .expect("Could not spawn Rita threads");
+    info!("Thread Spawner: {res:?}");
+
+    // Test for network convergence
+    test_reach_all(namespaces.clone());
+
+    test_routes(namespaces.clone(), expected_routes);
+
+    info!("Registering routers to the exit");
+    for r in namespaces.names.clone() {
+        if let NodeType::Client = r.node_type {
+            let res = register_to_exit(r.get_name()).await;
+            if !res.is_success() {
+                panic!("Failed to register {} to exit with {:?}", r.get_name(), res);
+            } else {
+                info!("{} registered to exit", r.get_name());
+            }
+        }
+    }
+
+    thread::sleep(Duration::from_secs(10));
+
+    let mut from_node: Option<Namespace> = None;
+    let mut forward_node: Option<Namespace> = None;
+    let mut end_node: Option<Namespace> = None;
+    // top up routers
+    for ns in namespaces.names.clone() {
+        info!("Namespace : {:?}", ns);
+        if ns.id == 1 {
+            from_node = Some(ns.clone());
+        }
+        if ns.id == 3 {
+            forward_node = Some(ns.clone());
+        }
+        if ns.id == 6 {
+            end_node = Some(ns.clone());
+        }
+    }
+
+    let mut to_topup = Vec::new();
+    for ident in rita_identities.client_identities {
+        to_topup.push(ident.get_althea_address());
+    }
+
+    info!(
+        "Sending aalthea to all nodes with address {:?}",
+        to_topup.clone()
+    );
+    send_althea_tokens(to_topup.clone()).await;
+
+    let balances = print_althea_balances(to_topup.clone(), "uUSDC".to_string()).await;
+    info!("USDC Balances are {:?}", balances);
+    let balances = print_althea_balances(to_topup, "aalthea".to_string()).await;
+    info!("Althea Balances are {:?}", balances);
+
+    //info!("After registering routers, sleeping for 30 sec");
+    //thread::sleep(Duration::from_secs(3000));
```

Comment:
maybe remove this commented code if it's not used?

Reply:
done